services/consensus_get_topic_info.proto

```diff
@@ -32,38 +39,50 @@ import "response_header.proto";
 import "consensus_topic_info.proto";
 
 /**
- * See [ConsensusService.getTopicInfo()](#proto.ConsensusService)
+ * Retrieve the latest state of a topic.
+ *
+ * This method is unrestricted and allowed on any topic by any payer
+ * account.<br/>
+ * A query for a deleted topic MAY succeed if the topic is within the
```

Comment:
A query for a deleted topic get INVALID_TOPIC_ID error code.

Reply:
As a matter of specification, if the topic is within the grace period for an autorenewal, it might not _actually_ be deleted yet. A compliant implementation _might_ choose to allow a query for such topics (even if the _current_ implementation does not).

Reply:
The current implementation does not query a deleted topic successfully. Not sure if these comments need to match the current implementation.

Reply:
The specification needs to be _compatible_ with current implementation (assuming no relevant bugs, of course).  It is acceptable to have a broader compatibility where appropriate.

Here, for instance, it seems reasonable to _permit_ a query to succeed in a particular case, while still allowing that the _current_ implementation rejects that query.

If you believe it is never reasonable to need to query topic information for a deleted (but not purged) topic, then I will rewrite to match.

Reply:
If you believe it's reasonable to permit a query to succeed in a particular case above which is not implemented yet. Maybe we should create an issue to track it will be implemented someday?

Reply:
I don't think it's necessary to create an issue.  Nothing in this specification indicates the current implementation _will_ (ever) permit such queries; it neither requires nor prohibits that option.

Not all requirements are absolute. The `MAY` keyword indicates that compliant implementations might, or might not, implement a particular option.